### PR TITLE
load test for migration

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: test/e2e-encryption/encryption_perf_test.go [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+		end := time.Now()
+		t.Logf("Populating etcd took %v", end.Sub(start))
+	}()
+
+	r := newRunner()
+	loaderFun := repeat(1, true,
+		createNamespace,
+		waitUntilNamespaceActive,
+		// TODO: consider adding ConfigMaps
+		repeatParallel(9010, 50, false, createSecret, reportSecret))
```
Comment (p0lyn0mial) [inline test/e2e-encryption/encryption_perf_test.go]: this is probably too much given the current speed of migration.

Comment (sttts) [reply test/e2e-encryption/encryption_perf_test.go]: how much time do we permit for e2e?

Comment (p0lyn0mial) [reply test/e2e-encryption/encryption_perf_test.go]: 4h for `e2e-encryption` tests and 1h for `standard-e2e` tests.

Do we want to have a separate job just for the load tests?

**Comment by p0lyn0mial** [comment]
This pull is ready for review, the test fails because I wanted to print the logs.

I think that we will create a separate CI job just for the performance test. That would give us isolation, speed up tests execution (we wouldn't have to wait for other tests to finish) and some room for the potential future changes.


the last test run gave me:
```
Verified 27995 Secrets 
Verified 15299 ConfigMaps 
migration took 24m3.550645976s 
```

File: pkg/operator/starter.go [hunk 2]
```diff
@@ -42,7 +42,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
+	dynamicClientConfig := dynamic.ConfigFor(ctx.KubeConfig)
+	dynamicClientConfig.Burst = 40
```
Comment (p0lyn0mial) [inline pkg/operator/starter.go]: we need to experiment and find good numbers for `QPS/Burst`

Comment (sttts) [reply pkg/operator/starter.go]: Please rename to `migrationClientConfig` and verify that it is only used for migration

Comment (sttts) [reply pkg/operator/starter.go]: Are those the numbers upstream migrator uses as well?

**Comment by p0lyn0mial** [comment]
will be merge as part of https://github.com/openshift/cluster-kube-apiserver-operator/pull/670

**Pull request closed without merging**